### PR TITLE
Ft-alerts-for-third-partys-155965587

### PR DESCRIPTION
**Pull request by Kalela**
# What does this PR do?
Allow the user to get alerts for third party apps.

### How should this be manually tested?
- Clone the repo into your local machine.
- cd into the project.
- set up the virtualenv and run ```pip install -r requirements.txt```.
- Run the tests using ```python manage.py test```
## OR
- Follow step 1 - 3
- Run the server using ```python manage.py runserver```
- Log in and head to integrations tab.
- Head to HealthWealth integration.
- Set your third party app url and get a check url from the checks.
- Click add integration and you're good to go.
- After creating an integration you can manually test it by clicking ```Run Check``.

### What are the relevant pivotal tracker stories?
Users should be able to create alerts for a series of third party online business automation applications .eg. shopify, dasheroo .etc.

#155965587

### CODE REVIEW

File: hc/api/tests/test_healthwealth.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from hc.api.models import ExternalChecks, Check
```
Comment (meshnesh) [inline hc/api/tests/test_healthwealth.py:1]: good approach in implementing the tests for the integration.

File: hc/api/tests/test_healthwealth.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class HealthWealthTestCase(BaseTestCase):
+    def setUp(self):
+        super(HealthWealthTestCase, self).setUp()
+        self.check = Check()
+        self.external_check = ExternalChecks()
+        self.external_check.save()
+
+    def test_user_can_create_integration(self):
```
Comment (meshnesh) [inline hc/api/tests/test_healthwealth.py:14]: it's good to implement doc string to state what it's supposed to do.

Comment (betsybeth) [inline hc/api/tests/test_healthwealth.py:14]: I think you forgot to add a docstring here..

File: templates/integrations/custom_integration_input.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
+{% load hc_extras humanize %}
```
Comment (meshnesh) [inline templates/integrations/custom_integration_input.html:1]: nice implementation and approach

File: hc/front/tests/test_healthwealth.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+class HealthWealthFrontTestCase(BaseTestCase):
+    def setUp(self):
+        """Set up the tests variables"""
+        super(HealthWealthFrontTestCase, self).setUp()
+        self.check = Check()
+        self.external_check = ExternalChecks()
+        self.external_check.save()
+
+    def test_it_renders_page(self):
```
Comment (betsybeth) [inline hc/front/tests/test_healthwealth.py:13]: Nice work in testing rendering page.

File: hc/front/views.py [hunk 5]
```diff
@@ -418,6 +428,27 @@ def add_pd(request):
...
+        form = ExternalChecksForm(data=request.POST)
+        if form.is_valid():
+            form.save()
+    else:
+        form = ExternalChecksForm()
+    external_checks = external_check()
+    ctx = {"form": form, "external_checks": external_checks}
+    return render(request, "integrations/add_healthwealth.html", ctx)
+
+def run_third_party_check(request, id):
```
Comment (meshnesh) [inline hc/front/views.py:444]: nice implemention

File: hc/front/views.py [hunk 6]
```diff
@@ -225,6 +227,14 @@ def remove_check(request, code):
 
     return redirect("hc-checks")
 
+@login_required
+def remove_integration(request, id):
```
Comment (meshnesh) [inline hc/front/views.py:231]: nice implementation

File: hc/front/tests/test_healthwealth.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
+from hc.api.models import ExternalChecks, Check
```
Comment (meshnesh) [inline hc/front/tests/test_healthwealth.py:1]: good implementation

**Review by meshnesh — APPROVED** [review]
Good approach in implementing the third party url's

File: hc/api/tests/test_healthwealth.py (hunk 1 above)
Comment (Kalela) [reply hc/api/tests/test_healthwealth.py:1]: Thanks

File: hc/api/tests/test_healthwealth.py (hunk 2 above)
Comment (Kalela) [reply hc/api/tests/test_healthwealth.py:14]: Yeah...lemme fix that.

File: templates/integrations/custom_integration_input.html (hunk 3 above)
Comment (Kalela) [reply templates/integrations/custom_integration_input.html:1]: Thank you.

File: hc/api/tests/test_healthwealth.py (hunk 2 above)
Comment (Kalela) [reply hc/api/tests/test_healthwealth.py:14]: Thank you. Lemme fix it now.

**Comment by FelixWambiri** [comment]
Good implementation especially nice work on the tests.

**Pull request merged**